docs/content/development/extensions-core/first-last-string.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This file's in the extensions-core folder, but the link to it in extensions.md points at extensions-contrib.

Reply:
You are right!